src/hotspot/share/utilities/elfFile.hpp

```diff
@@ -483,7 +483,7 @@ class DwarfFile : public ElfFile {
 
     DwarfFile* _dwarf_file;
     MarkedDwarfFileReader _reader;
-    uint32_t _section_start_address;
+    uint64_t _section_start_address;
```

Comment:
This is platform-specific.
```suggestion
    uintptr_t _section_start_address;
```